packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/template.html

```diff
@@ -1,4 +1,7 @@
-<div class="pci-new-project-promotion-banner" data-ng-if="$ctrl.voucher.valid">
+<div
+    class="pci-new-project-promotion-banner"
+    data-ng-if="$ctrl.isVoucherValid($ctrl.voucher)"
```

Comment:
you can use `this.voucher` directly in the controller instead of adding arguments